index.html

```diff
@@ -1,5 +1,6 @@
 <!DOCTYPE html>
 <html lang = "en">
+    <!-- a change to make a pull request -->
```

Comment:
We will be discussing pull requests in class.  This is so you have one to play with.  Most people have corrections in their pull requests, but your code is fine as it is!